### ISSUE TITLE
When using `annotate: true`, test summary not shown from 1.9-beta3 onward

### ISSUE THREAD

**Issue by simsurace** [julia-actions/julia-runtest#78]
May be related to #76, but I finally found why my test summaries were never shown on newer Julia versions.

**Comment by ericphanson** [comment]
Yes, that's the same issue. https://github.com/julia-actions/julia-runtest/pull/79 will disable `annotate: true` except on 1.8 to workaround this issue
